udata/templates/dataset/resource/list-item.html

```diff
@@ -28,7 +33,7 @@ <h4 class="list-group-item-heading ellipsis">
             {% set edit_path = 'dataset/{id}/resource/{rid}' %}
         {% endif %}
         <div class="btn-group btn-group-xs tools">
-            <a class="btn btn-default btn-edit" v-tooltip title="{{ _('Edit') }}"
+            <a @click.stop class="btn btn-default btn-edit" v-tooltip title="{{ _('Edit') }}"
```

Comment:
OMG, is this working ? If so I understand the above removal !!

Reply:
Yep, it should work ;-) At least it does on my machine.